metadata/src/main/java/edu/unc/lib/dl/util/PremisEventBuilder.java

```diff
@@ -0,0 +1,93 @@
+package edu.unc.lib.dl.util;
+
+import java.text.MessageFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.ModelFactory;
+import com.hp.hpl.jena.rdf.model.Resource;
+
+import edu.unc.lib.dl.event.PremisLogger;
+import edu.unc.lib.dl.rdf.Premis;
+
+public class PremisEventBuilder {
+	private String eventId;
+	private Resource eventType;
+	private Date date;
+	private Model model;
+	private PremisLogger premisLogger;
+	private Resource premisObjResc;
+	
+	public PremisEventBuilder(String eventId, Resource eventType, Date date,
+			PremisLogger premisLogger) {
+		this.eventId = eventId;
+		this.eventType = eventType;
+		this.date = date;
```

Comment:
Doesn't seem like there is a reason to hold date and eventType as instance variables on the PremisEventBuilder class, they don't need to live long enough for the call to addEvent, so maybe just pass them in as parameters?